components/centraldashboard/public/components/dashboard-view.pug

```diff
@@ -1,44 +1,47 @@
-.column
-    paper-card#Quick-Links(heading='Quick shortcuts')
-        template(is='dom-repeat', items='[[quickLinks]]')
-            iframe-link.link(role='link', href$="[[item.link]]")
-                paper-icon-item
-                    iron-icon(icon='kubeflow:bolt', slot='item-icon')
-                    paper-item-body(two-line)
-                        .header [[item.text]]
-                        aside(secondary) [[item.desc]]
-                    paper-ripple
-template(is='dom-if', if='[[platformDetails.resourceChartsLink]]')
+div#grid
     .column
-        resource-chart(header-text='Cluster CPU Utilization', metric='cpu',
-            interval='Last60m',
-            external-link='[[platformDetails.resourceChartsLink]]',
-            external-link-text='[[platformDetails.resourceChartsLinkText]]')
-        resource-chart(header-text='Pod CPU Utilization', metric='podcpu',
-            interval='Last60m',
-            external-link='[[platformDetails.resourceChartsLink]]'
-            external-link-text='[[platformDetails.resourceChartsLinkText]]')
-.column
-    template(is='dom-if', if='[[platformDetails.links]]')
-        paper-card#Platform-Links(class$='[[platformDetails.name]]',
-            heading='[[platformDetails.title]]', image='[[platformDetails.logo]]')
-            template(is='dom-repeat', items='[[platformDetails.links]]')
-                a.link(href$='[[item.link]]', tabindex='-1', target='_blank')
+        paper-card#Quick-Links(heading='Quick shortcuts')
+            template(is='dom-repeat', items='[[quickLinks]]')
+                iframe-link.link(role='link', href$="[[item.link]]")
```

Comment:
Instead of providing role here, can we have the `role` defined from within the component, defaulted to `link`?

Reply:
Good call. For some reason the idea of setting the attribute in the component's setup eluded me. Thanks for the suggestion!